Release job fails

- **Version**: 2.2.0-RC1

https://travis-ci.org/scalameta/scalafmt/jobs/599254480

## Problem

`sbt ci-release docs/docusaurusPublishGhpages` command fails.

```scala
[error] (run-main-0) java.lang.NoClassDefFoundError: scala/tools/nsc/reporters/AbstractReporter
[error] java.lang.NoClassDefFoundError: scala/tools/nsc/reporters/AbstractReporter
[error] 	at java.lang.ClassLoader.defineClass1(Native Method)
[error] 	at java.lang.ClassLoader.defineClass(ClassLoader.java:763)
[error] 	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
[error] 	at java.net.URLClassLoader.defineClass(URLClassLoader.java:468)
[error] 	at java.net.URLClassLoader.access$100(URLClassLoader.java:74)
[error] 	at java.net.URLClassLoader$1.run(URLClassLoader.java:369)
[error] 	at java.net.URLClassLoader$1.run(URLClassLoader.java:363)
[error] 	at java.security.AccessController.doPrivileged(Native Method)
[error] 	at java.net.URLClassLoader.findClass(URLClassLoader.java:362)
[error] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[error] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
[error] 	at mdoc.internal.markdown.MarkdownCompiler$.fromClasspath(MarkdownCompiler.scala:91)
[error] 	at mdoc.internal.cli.Settings.validate(Settings.scala:177)
[error] 	at mdoc.internal.cli.MainOps$.$anonfun$process$1(MainOps.scala:224)
[error] 	at metaconfig.Configured.andThen(Configured.scala:30)
[error] 	at mdoc.internal.cli.MainOps$.process(MainOps.scala:224)
[error] 	at mdoc.Main$.process(Main.scala:29)
[error] 	at docs.Main$.main(Main.scala:28)
[error] 	at docs.Main.main(Main.scala)
[error] 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[error] 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[error] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[error] 	at java.lang.reflect.Method.invoke(Method.java:498)
[error] Caused by: java.lang.ClassNotFoundException: scala.tools.nsc.reporters.AbstractReporter
[error] 	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
[error] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[error] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
[error] 	at java.lang.ClassLoader.defineClass1(Native Method)
[error] 	at java.lang.ClassLoader.defineClass(ClassLoader.java:763)
[error] 	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
[error] 	at java.net.URLClassLoader.defineClass(URLClassLoader.java:468)
[error] 	at java.net.URLClassLoader.access$100(URLClassLoader.java:74)
[error] 	at java.net.URLClassLoader$1.run(URLClassLoader.java:369)
[error] 	at java.net.URLClassLoader$1.run(URLClassLoader.java:363)
[error] 	at java.security.AccessController.doPrivileged(Native Method)
[error] 	at java.net.URLClassLoader.findClass(URLClassLoader.java:362)
[error] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[error] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
[error] 	at mdoc.internal.markdown.MarkdownCompiler$.fromClasspath(MarkdownCompiler.scala:91)
[error] 	at mdoc.internal.cli.Settings.validate(Settings.scala:177)
[error] 	at mdoc.internal.cli.MainOps$.$anonfun$process$1(MainOps.scala:224)
[error] 	at metaconfig.Configured.andThen(Configured.scala:30)
[error] 	at mdoc.internal.cli.MainOps$.process(MainOps.scala:224)
[error] 	at mdoc.Main$.process(Main.scala:29)
[error] 	at docs.Main$.main(Main.scala:28)
[error] 	at docs.Main.main(Main.scala)
[error] 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[error] 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[error] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[error] 	at java.lang.reflect.Method.invoke(Method.java:498)
[error] Nonzero exit code: 1
[error] (docs / Compile / run) Nonzero exit code: 1
[error] Total time: 7 s, completed Oct 17, 2019 5:17:16 PM
The command "sbt ci-release docs/docusaurusPublishGhpages" exited with 1.
```

## Notes 

We can't release new versions :(

--- a/project/plugins.sbt
+++ b/project/plugins.sbt
@@ -4,7 +4,7 @@ resolvers ++= Seq(
   Resolver.bintrayIvyRepo("jetbrains", "sbt-plugins")
 )
 
-addSbtPlugin("org.scalameta" % "sbt-mdoc" % "1.3.5")
+addSbtPlugin("org.scalameta" % "sbt-mdoc" % "1.3.6")
 addSbtPlugin("com.geirsson" % "sbt-scalafmt" % "1.6.0-RC4")
 addSbtPlugin("com.geirsson" % "sbt-ci-release" % "1.4.31")
 addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.7.0")

